docs/reference/commandline/plugin_rm.md

```diff
@@ -12,20 +12,22 @@ parent = "smn_cli"
 # plugin rm (experimental)
 
 ```markdown
-Usage:  docker plugin rm PLUGIN
+Usage:  docker plugin rm [OPTIONS] PLUGIN [PLUGIN...]
 
 Remove a plugin
 
 Aliases:
   rm, remove
 
 Options:
-      --help   Print usage
+      -f, --force  Force the removal of an active plugin
+      --help       Print usage
 ```
 
 Removes a plugin. You cannot remove a plugin if it is active, you must disable
 a plugin using the [`docker plugin disable`](plugin_disable.md) before removing
-it.
+it (or use --force, use of force is not recommended, since it can affect
```

Comment:
Its a side effect of the disable, rather than the remove itself. We should move the warning to plugin disable's documentation. For --force, we can say that it disables the plugin before removing.